dev-requirements.txt

```diff
@@ -20,6 +20,9 @@ sphinx-rfcsection~=0.1.1
 # use fork of vcrpy 5.x until kevin1024/vcrpy#777 is (hopefully) accepted
 # (or until py3.9 EOL... in 10/2025, I HOPE NOT)
 vcrpy @ git+https://github.com/sopel-irc/vcrpy@uncap-urllib3
+# also have to use a version of urllib3 that doesn't use the `version_string`
+# attr of the response object, because vcrpy won't support it until 7.x
```

Comment:
We may as well refer to #2647 here too, I think?

Reply:
Hmm, it could but that was the PR that added the comment in question. I will add a reference to #2456.

Reply:
f03fbae1aacd653f15e0bbc0b520d59c0d64b09e - cherry-picked to #2655 rather than creating a whole new PR for it.